<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>183</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>World size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="worldSize_label">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="worldSize_horizontalSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Initial % rabbits:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="initialRabbit_label">
         <property name="text">
          <string>20</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="initialRabbit_horizontalSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Initial % foxes:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="initialFox_label">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="initialFoxes_horizontalSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Breeding rabbits:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="breedingRabbit_label">
         <property name="text">
          <string>0.04</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="breedingRabbits_horizontalSlider">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Breeding foxes:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="breedingFox_label">
         <property name="text">
          <string>0.8</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="breedingFoxes_horizontalSlider">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Mortality foxes:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mortalityFox_label">
         <property name="text">
          <string>0.15</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="mortalityFoxes_horizontalSlider">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Effectiveness foxes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="effectivenessFox_label">
         <property name="text">
          <string>0.7</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="effectivenessFoxes_horizontalSlider">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>Simulation speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="simulationSpeed_label">
         <property name="text">
          <string>0.5</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="simulationSpeed_horizontalSlider">
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>181</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="buttons_formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="start_pushButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="step_pushButton">
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="reset_pushButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>711</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QWidget" name="view_widget" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>30</y>
      <width>261</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="chart1_widget" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="chart2_widget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
